java/configuration/src/main/java/sleeper/configuration/properties/instance/CommonProperty.java

```diff
@@ -161,12 +161,12 @@ public interface CommonProperty {
             .runCdkDeployWhenChanged(true).build();
     UserDefinedInstanceProperty METRICS_LAMBDA_CONCURRENCY_RESERVED = Index.propertyBuilder("sleeper.metrics.concurrency.reserved")
             .defaultProperty(DEFAULT_LAMBDA_CONCURRENCY_RESERVED)
-            .description("The reserved concurrency for the table metrics lambda.\n" +
+            .description("Default value for the reserved concurrency for each lambda within the Sleeper instance..\n" +
                     "See reserved concurrency overview at: https://docs.aws.amazon.com/lambda/latest/dg/configuration-concurrency.html")
             .propertyGroup(InstancePropertyGroup.COMMON).build();
     UserDefinedInstanceProperty METRICS_LAMBDA_CONCURRENCY_MAXIMUM = Index.propertyBuilder("sleeper.metrics.concurrency.max")
             .defaultProperty(DEFAULT_LAMBDA_CONCURRENCY_MAXIMUM)
-            .description("The maximum given concurrency allowed for the table metrics lambda.\n" +
+            .description("Default value for the maximum concurrency assigned to each lambda within the Sleeper instance.\n" +
```

Comment:
Looks like you meant to change the default property rather than the table metrics property?